client/extensions/wp-job-manager/state/settings/test/reducer.js

```diff
@@ -0,0 +1,197 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import { useSandbox } from 'test/helpers/use-sinon';
+import {
+	WP_JOB_MANAGER_FETCH_ERROR,
+	WP_JOB_MANAGER_FETCH_SETTINGS,
+	WP_JOB_MANAGER_UPDATE_SETTINGS,
+} from '../../action-types';
+import { DESERIALIZE, SERIALIZE } from 'state/action-types';
+import { fetching, items } from '../reducer';
+
+describe( 'reducer', () => {
+	const primarySiteId = 123456;
+	const secondarySiteId = 456789;
+
+	useSandbox( ( sandbox ) => {
+		sandbox.stub( console, 'warn' );
+	} );
+
+	describe( 'fetching()', () => {
+		const previousState = deepFreeze( {
+			[ primarySiteId ]: true,
+		} );
+
+		it( 'should default to an empty object', () => {
+			const state = fetching( undefined, {} );
+
+			expect( state ).to.eql( {} );
+		} );
+
+		it( 'should set request to true if settings are being fetched', () => {
```

Comment:
Shouldn't that be `state` instead of `request`? (Also a couple more instances below.)